### PR TITLE
ACUTATORS: fix actuator driver, add to sim, add actuator proxy

### PR DESCRIPTION
**Pull request by kev-the-dev**
* fix bug in acutator_driver using old alarm system
* added actuator_sim back to duck.launch, so missions using actuators won't fail
* added actuator proxy to sub singleton to make it easier for missions to move acutators
* added actuator_test mission as both a test for the actuator_proxy and a reference for missions using actuators

### CODE REVIEW

**Comment by Grymessa** [comment]
The subc teleport list looks great, that was a good idea.

File: command/sub8_missions/sub8/sub_singleton.py [hunk 1]
```diff
@@ -176,6 +247,8 @@ def _init(self):
         self._tf_listener = yield tf.TransformListener(self.nh)
 
         self.vision_proxies = _VisionProxies(self.nh, 'vision_proxies.yaml')
+        self.passive_sonar = _PassiveSonarProxy(self.nh, 'hydrophones')
```
Comment (DSsoto) [inline command/sub8_missions/sub8/sub_singleton.py]: The namespace should be `/passive_sonar`

File: command/sub8_missions/sub8/sub_singleton.py [hunk 2]
```diff
@@ -161,6 +164,74 @@ def go_trajectory(self, *args, **kwargs):
...
+        return self.set('gripper', True)
+
+    def gripper_close(self):
+        return self.set('gripper', False)
+
+    def shoot_torpedo1(self):
+        return self.set('torpedo1', True)
+
+    def shoot_torpedo2(self):
+        return self.set('torped2', True)
```
Comment (DSsoto) [inline command/sub8_missions/sub8/sub_singleton.py]: typo?

**Review by DSsoto — CHANGES_REQUESTED** [review]
Have you tested the proxys at all?

**Comment by kev-the-dev** [comment]
I think I'm going to separate the teleport changes to yet another PR, because the passive sonar proxy will break missions until passive sonar OS is merged.

**Comment by DSsoto** [comment]
Good idea.

**Comment by kev-the-dev** [comment]
EDITS:
* put the TP stuff in a different PR as it's unrelated #291 
* removed passive sonar proxy, as it will break things until PassiveSonarOS is merged. I still have it [on this branch](https://github.com/ironmig/SubjuGator/blob/passive-sonar/command/sub8_missions/sub8/sub_singleton.py)
* got the actuator driver up and running again in SIM, and a mission to test the actuator proxy

File: command/sub8_missions/missions/test_actuators.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        failed = True
+
+    test_runner = test()
+    test_runner.addErrback(err)
+    yield test_runner
+    global failed
+    if not failed:
+        fprint('Success with all actuators!', msg_color='blue')
+    else:
+        fprint('Failed 1 or more actuator calls', msg_color='red')
```
Comment (DSsoto) [inline command/sub8_missions/missions/test_actuators.py:34]: This is not as usefull if it doesn't say what actuator call failed

**Review by DSsoto — APPROVED** [review]

File: command/sub8_missions/missions/test_actuators.py (hunk 3 above)
Comment (kev-the-dev) [reply command/sub8_missions/missions/test_actuators.py:34]: It will say where it failed in the errback when the service fails.

**Pull request merged**